source/ranges.tex

```diff
@@ -2387,6 +2391,8 @@
     friend constexpr sentinel_t<R> end(@\placeholder{ref-view}@ r)
     { return r.end(); }
   };
+  template<class R>
+    ref_view(R&) -> ref_view<R>;
```

Comment:
These should technically be `\placeholder{ref-view}`s in this PR, but motions-2019-lwg-16 is renaming *`ref-view`* to `ref_view` so this is the proper final state for this line post-Kona. Maybe the rename here should be in a separate post-merge commit?

Reply:
Since the motions will all be merged at the same time, I think it's OK to leave it as is.  Less headaches for Richard :).

BTW, are you reviewing all the applied issues in this motion's PR?  If so, I'd like to assign you as the reviewer (to prevent someone else from doing a full review and duplicating your work).

Reply:
I looked over all the changes to make sure there was nothing insane, but I haven't compared with the ingress paper.

Reply:
Should this PR be made red given that LWG3173 (in it's current state) depends on LWG motion 16 which is also red?

Reply:
I've changed this to *`ref-view`* for now. I'll keep this in mind when applying LWG Motion 16.